app/scrapers/yahoo.py

```diff
@@ -34,10 +34,10 @@ def results_search(self,query):
                 r = y.get('href')
```

Comment:
This is a PR for quora. This change should come in another PR.

Reply:
This is not a change. I was trying to fix merge conflicts, that's why I made this change.